lib/OfxParser/Parser.php

```diff
@@ -52,18 +50,17 @@ public function loadFromString($ofxContent)
     }
 
     /**
-     * Detect if the OFX file is on one line. If it is, add newlines automatically.
+     * Prepare OFX file contents.
      *
      * @param string $ofxContent
      * @return string
      */
-    private function conditionallyAddNewlines($ofxContent)
+    private function fixNewlines($ofxContent)
     {
-        if (preg_match('/<OFX>.*<\/OFX>/', $ofxContent) === 1) {
-            return str_replace('<', "\n<", $ofxContent); // add line breaks to allow XML to parse
-        }
-
-        return $ofxContent;
+        // clear all new line characters first
+        $ofxContent = str_replace(["\r", "\n"], '', $ofxContent);
```

Comment:
Hmm, seems unlikely, but possible that there might be other newlines we didn't intend to replace here (for example, maybe a transaction description has a newline that should be kept for whatever reason). We don't have an existing test, but this *may* be a BC break.

Reply:
understood